app/src/main/java/com/dsd/carcompanion/HomeFragment.kt

```diff
@@ -0,0 +1,95 @@
+package com.dsd.carcompanion
+
+import android.os.Bundle
```

Comment:
This file should go in its own folder, even if it is alone there

Reply:
done